src/asmcnc/keyboard/custom_keyboard.py

```diff
@@ -22,12 +23,17 @@ def __init__(self, **kwargs):
         reload(sys)
         sys.setdefaultencoding('utf-8')
 
+        self.bind(layout=self.set_keyboard_background)
+        self.bind(do_translation=self.set_keyboard_background)
+
         self.text_instance = None
 
-        self.kr_layout = "./asmcnc/keyboard/layouts/kr.json"
-        self.numeric_layout = "./asmcnc/keyboard/layouts/numeric.json"
+        dirname = os.path.dirname(__file__)
+
+        self.kr_layout = os.path.join(dirname, "layouts/kr.json")
+        self.numeric_layout = os.path.join(dirname, "layouts/numeric.json")
         self.qwerty_layout = "data/keyboards/qwerty.json"
-        self.qwertyKR_layout = "./asmcnc/keyboard/layouts/qwertyKR.json"
+        self.qwertyKR_layout = os.path.join(dirname, "layouts/qwertyKR.json")
```

Comment:
don't include '/' in the path, use separate arguments inside os.path.join, as it will use the correct separator for the current OS

os.path.join(dirname, "layouts", "qwertyKR.json")

or even have layouts_dir = os.path.join(dirname, "layouts") then os.path.join(layouts_dir, "qwertyKR.json")